package.json

```diff
@@ -32,21 +32,27 @@
     "url": "https://github.com/CrowdStrike/eslint-config-crowdstrike-node/issues"
   },
   "homepage": "https://github.com/CrowdStrike/eslint-config-crowdstrike-node#readme",
+  "dependencies": {
+    "@eslint/eslintrc": "3.1.0",
+    "@eslint/js": "9.8.0",
+    "globals": "15.8.0"
+  },
   "devDependencies": {
     "@crowdstrike/commitlint": "^8.0.0",
-    "eslint": "^8.0.0",
+    "eslint": "^9.0.0",
     "eslint-config-crowdstrike": "10.1.0",
     "eslint-config-crowdstrike-node": "link:",
-    "eslint-plugin-json-files": "^4.0.0",
+    "eslint-plugin-json-files": "^4.3.0",
     "eslint-plugin-n": "^17.0.0",
     "remark-cli": "^12.0.0",
     "remark-preset-lint-crowdstrike": "^4.0.0",
     "renovate-config-standard": "^2.0.0",
     "standard-version": "^9.0.0"
   },
   "peerDependencies": {
+    "eslint": ">=8.57.0",
     "eslint-config-crowdstrike": ">=1",
-    "eslint-plugin-json-files": ">=0.8",
+    "eslint-plugin-json-files": ">=4.3.0",
```

Comment:
pretty sure this is unncessary, as the same rule exists in `.8`

Reply:
I'm fine with this.